calcom/cal.com
# feat: adding Bolna Voice AI integration for cal.com

## What does this PR do?



- Adds the [Bolna AI](https://playground.bolna.dev/auth/calcom) integration for cal.com.



## Screenshots
<img width="1597" alt="Screenshot 2024-09-26 at 19 12 50" src="https://github.com/user-attachments/assets/96215467-466b-406e-8e1e-a1a6db53e19d">

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com/docs). N/A

## How should this be tested?



- Are there environment variables that should be set? n/a
- What are the minimal test data to have? - have a local setup and navigate to http://localhost:3000/apps/bolna for installation
